src/main/scala/util/GeneratorUtils.scala

```diff
@@ -139,7 +139,12 @@ trait GeneratorApp extends App with HasGeneratorUtilities {
 
 object ElaborationArtefacts {
   var files: Seq[(String, () => String)] = Nil
-  def add(extension: String, contents: => String) {
+
+  def add(extension: String, contents: => String) = {
```

Comment:
Are you intentionally making this no longer a `Unit` function with `=`?

Reply:
no. thanks.